src/pages/NewUser/helper.tsx

```diff
@@ -16,13 +19,22 @@ export const newUserFormProps: FormProps = {
 	method: METHOD.POST,
 	inputs: [
 		{
-			id: 'name',
+			id: 'first name',
```

Comment:
`id` named exactly the same as the field doesn't seem good to me. you can do the same as you sis in another file and leave something like `01`, `02`, etc.

Reply:
+